src/protocolsupport/protocol/packet/middle/clientbound/play/MiddleSpawnLiving.java

```diff
@@ -23,9 +23,9 @@ public MiddleSpawnLiving(ConnectionImpl connection) {
 	protected double x;
 	protected double y;
 	protected double z;
-	protected int yaw;
-	protected int pitch;
-	protected int headPitch;
+	protected byte yaw;
```

Comment:
and again here and above- using signed byte rotations as we dont need to worry about wrapping when adding or subtracting rotation